packages/docs/docs/auth/authentication-methods/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Authentication Methods",
```

Comment:
The `/auth/authentication-methods/...` URLs look a bit chunky to me, would we want to slim down to e.g. `methods` or `integrations`?

Reply:
Good point re: URL. 

Will change to `/methods`

Reply:
Will resolve titles in a future PR, since there's a bit of article renaming that needs to be done